### PR TITLE
ONNX ppo inference on MuJoCo wasm

### PR DESCRIPTION
**Pull request by WT-MM**
**What does this PR do?**

Adds logic to run a standing policy on MuJoCo WASM using ONNX

**What issues does this PR fix or reference?**

Continuation of #419

### CODE REVIEW

**Comment by budzianowski** [comment]
Please use the latest policy from the sim repo. It works at 50 Hz so you have to adjust the decimation parameters here.

File: frontend/public/examples/scenes/dora2.xml [hunk 1]
```diff
@@ -0,0 +1,269 @@
+<mujoco model="Dora2_yin">
```
Comment (budzianowski) [inline frontend/public/examples/scenes/dora2.xml:1]: Remove all dora files

**Review by budzianowski — COMMENTED** [review]
Remove policy_1.onnx file and other unused ones.

**Review by Winston-Hsiao — CHANGES_REQUESTED** [review]
@WT-MM Looks good, but can you check the github actions workflow and resolve the linting errors yourself? I can't clone/pull your branch/fork because of conflicts, and I'm trying to finish up kernel image download.

**Review by Winston-Hsiao — APPROVED** [review]
LGTM

**Pull request merged**